common/data_refinery_common/migrations/0038_paginate_default_svd_algorithm.py

```diff
@@ -1,26 +1,36 @@
 from django.db import migrations, models
-from django.core.paginator import Paginator
+
+def batch_update(queryset, batch=1000, **changes):
+    """ Update per batch """
+    is_done = False
+    current = 1
+
+    while not is_done:
+        start = (current - 1) * batch
+        end = start + batch
+        pks = queryset.values_list('pk', flat=True)[start:end][::1]
```

Comment:
Should this be: `queryset[start:end].values_list('pk', flat=True)`? `values_list` will return a list with every id.

Reply:
my understanding is that the slice is what executes the queryset but with a limit offset

Reply:
so it would be the same

Reply:
At the top of this page it explains which operations execute the query list.

https://docs.djangoproject.com/en/2.2/ref/models/querysets/#values-list